### ISSUE TITLE
Support css media Queries

### ISSUE THREAD

**Issue by mattiasmak** [Juicy/juicy-tile-editor#151]
The icons with 320, 480, 960 and 1200px. Isn't that supposed to function similar to css media queries? Right now it only sets min-width: and that seems very wrong. 

Am I missing something?

**Comment by mattiasmak** [comment]
Ok. We have a big problem with the min-width:1200px being set everywhere. How should we handle that?

**Comment by miyconst** [comment]
@mattiasmak you are right. Those icons suppose to be similar to css media queries, good that it feels that way, but unfortunately it is not supported yet.

cc @tomalec

**Comment by mattiasmak** [comment]
If it's not supported it would be great if you stop setting min-width. Now it defaults to 1200px and messes up our layout everywhere and all the time.

Is that possible?

**Comment by tomalec** [comment]
@miyconst do you remember why you did https://github.com/StarcounterPrefabs/launcher-include/commit/31ad5f6c7fd94570da1270dc81e49153eb1d6538 ?

**Comment by miyconst** [comment]
I do, David complained about non responsive layout. Means, once page loaded it can't be resized to a smaller size. That is why I set the default size to be `1200px`, but looks like this is even worse.

**Comment by tomalec** [comment]
o before you were not able to shrink it below first size (which could be at leas somehow applicable),
and right now we cannot shrink it, or render it below 1200px, what is usually what we do want at all.

**Comment by warpech** [comment]
I have the same problem in Chatter. Every nested partial wants to be 1200px wide, which doesn't make sense when I put it to a 500px wide group.

@miyconst could you please open a task about fixing this "hardcoded 1200px problem" introduced by https://github.com/StarcounterPrefabs/launcher-include/commit/31ad5f6c7fd94570da1270dc81e49153eb1d6538? I understand that it was solving a real issue, so it should include the description of the real issue. Then I propose that @tomalec prepares a better fix.

Thanks!

**Comment by miyconst** [comment]
The new issue with some description is available here: https://github.com/Juicy/juicy-tile-editor/issues/154.

**Comment by tomalec** [comment]
The issue on ranges support for `juicy-tile-*` is here https://github.com/Juicy/juicy-tile-table/issues/9
